.github/workflows/build.yml

```diff
@@ -80,11 +80,27 @@ jobs:
       - uses: hashicorp/setup-terraform@v2
         with:
           terraform_version: ${{ steps.setup-env.outputs.terraform-version }}
+      # Begin Go packages install below
```

Comment:
Now that we've decided to go with alphabetical order (per @mcdonnnj's request) this comment should be removed:
```suggestion
```